config/mainmenu.json

```diff
@@ -2,7 +2,11 @@
 	//images used in game selection screen
 	"game-select" : ["gamselb0", "gamselb1"],
 
-	"loading" : ["loadbar"],
+	"loading" :
+	{
+		"background" : ["loadbar"],
+		"x": 395, "y": 548, "size": 18, "amount": 20, "name": "loadprog"
+	},
```

Comment:
We really need to find a way to detect outdated mods without explicit "compatibility" field in every mod...

Reply:
backward compatibility is preserved. If mod is not updated, there will be only loading screen without progress bar